.travis.yml

```diff
@@ -1,10 +1,12 @@
 ---
 language: python
-python: "2.7.15"
+python: "3.9"
 sudo: required
 
 env:
   - PIPENV_IGNORE_VIRTUALENVS=1
+  - MOLECULE_DISTRO=debian:buster-slim
+  - MOLECULE_DISTRO=debian:bullseye-slim
 
 services:
   - docker
```

Comment:
I think build is failing because the `install` section, as it installs from test-requirements.txt. I would copy-paste it from another repo